src/routes/onboarding/add-to-homescreen.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import smRectangle from "../../images/Smallrectangle.svg";
+import shareBar from "../../images/Sharebar.svg";
+import lgRectangle from "../../images/Largerectangle.svg";
+import icons from "../../images/Addtohomescreenicons.svg";
+
+export const AddToHomescreen: React.FC = () => {
+  return (
+    <div className="flex flex-col bg-gradient-to-b from-[#F5F5F5] to-[#9ECCFA] overflow-auto max-w-screen-md pb-8 relative m-auto h-screen">
+      <h2 className="font-text text-section-header font-medium mt-5 leading-[52.8px] text-center max-w-[358px] mx-auto mb-8">

```

Comment:
the header's `line-height` is incorrect, it should be 38.4px according to the figma.